.rubocop.yml

```diff
@@ -128,6 +128,9 @@ Rails/ReversibleMigration:
   Exclude:
        - 'db/migrate/20150111085317_polymorph_asset.rb'
 
+Layout/SpaceBeforeBlockBraces:
+  Enabled: false
```

Comment:
I would not add this, we have spaces everywhere and removing empty block in `policies_helper` should take care of the last offence